src/lib/auth/account.ts

```diff
@@ -1,19 +1,29 @@
 import * as webnative from 'webnative'
 import type FileSystem from 'webnative/fs/index'
+import { get as getStore } from 'svelte/store'
 
 import { asyncDebounce } from '$lib/utils'
-import { filesystemStore, sessionStore } from '../../stores'
+import { filesystemStore, programStore, sessionStore } from '../../stores'
 import { getBackupStatus } from '$lib/auth/backup'
 import { ACCOUNT_SETTINGS_DIR } from '$lib/account-settings'
 import { AREAS } from '$routes/gallery/stores'
 import { GALLERY_DIRS } from '$routes/gallery/lib/gallery'
 
+
 export const isUsernameValid = async (username: string): Promise<boolean> => {
-  return webnative.account.isUsernameValid(username)
+  const program = getStore(programStore)
+  return program.auth[ webnative.strategyTypes.default ].isUsernameValid(username)
```

Comment:
Not a huge fan of this `program.auth[ webnative.strategyTypes.default ]` part, appears in multiple places in the app. I guess you could create a shortcut in a store with the name `authStrategy`, but open to other suggestions.

Reply:
We can also just use `program.auth.webCrypto`, but that makes it slightly harder to switch to another auth strategy (in case you want to leave that open).